src/main/java/org/opensearch/security/auth/UserInjector.java

```diff
@@ -137,7 +137,7 @@ public InjectedUser getInjectedUser() {
 
         // custom attributes
         if (parts.length > 3 && !Strings.isNullOrEmpty(parts[3])) {
-            Map<String, String> attributes = SecurityUtils.mapFromArray((parts[3].split(",")));
```

Comment:
Similar to above, are the refactors required for the compilation warnings? 

I don't have a personal problem with doing two things at once, but maybe we can modify the description so we know when these were made in the future? I think only the description is auto picked up.

Reply:
Yes. The problem is that if we just print warning nobody will fix them :-). There are a  lot of warnings in the OS code and nobody has addressed them yet.